test/ExtensibleEvents.test.ts

```diff
@@ -49,9 +49,11 @@ describe('ExtensibleEvents', () => {
                     [M_TEXT.name]: "Hello World",
                 },
             };
-            const event = (new ExtensibleEvents()).parse(input);
+            const event: ExtensibleEvent = (new ExtensibleEvents()).parse(input);
             expect(event).toBeDefined();
             expect(event instanceof MessageEvent).toBe(true);
+            expect(event.serialize().type).toBe("m.room.message");
+            expect(event.serialize().content).toHaveProperty("body", "Hello World");
```

Comment:
This sort of testing is covered by other tests?

Reply:
ah, this isn't trying to test that it serializes back into an appropriate shape, just that the parser produces a sensible event object. The actual parsing is tested by `MessageEvent`'s tests.

Reply:
Replaced with a cast to MessageEvent and then checking it actually contains `Hello World`